apps/api/v2/src/ee/bookings/2024-08-13/services/input.service.ts

```diff
@@ -179,8 +179,10 @@ export class InputBookingsService_2024_08_13 {
     // note(Lauris): timeBetween 0=yearly, 1=monthly and 2=weekly
     const timeBetween = occurrence.freq;
 
-    if (!repeatsTimes) {
-      throw new Error("Repeats times is required");
+    if (inputBooking.recurrenceCount && inputBooking?.recurrenceCount > occurrence.count) {
```

Comment:
NIT: no need for ? operator

Reply:
Well spotted - removed in [Update input.service.ts](https://github.com/calcom/cal.com/pull/16989/commits/8bd4d0fec067913c61023b40cd3c56baa4631e45)